### ISSUE TITLE
wrong url generated for AZURE openai 

### ISSUE THREAD

**Issue by zivshapiraintel** [continuedev/continue#982]
### Before submitting your bug report

- [X] I believe this is a bug. I'll try to join the [Continue Discord](https://discord.gg/NWtdYexhMs) for questions
- [X] I'm not able to find an [open issue](https://github.com/continuedev/continue/issues?q=is%3Aopen+is%3Aissue) that reports the same bug
- [X] I've seen the [troubleshooting guide](https://continue.dev/docs/troubleshooting) on the Continue Docs

### Relevant environment info

```Markdown
- OS:Windows 11
- Continue: 0.9.88 (pre release)
- IDE:VS Code v1.87.2
```

### Description

before this version our apiBase was completed correctly by the plugin: `our apiBase`/openai/deployments/gpt-4/chat/completions?api-version=2024-02-15-preview, but now it is completed without a / between gpt-4 and chat, so we get an error `Continue error: HTTP 404 Not Found from  <our apiBase>/openai/deployments/gpt-4chat/completions?api-version=2024-02-15-preview`

### To reproduce

connect via the plugin to azure (...openai.azure.com) and then upgrade to the release above

**Comment by elabbarw** [comment]
falling back to v0.8.17 works

**Comment by elabbarw** [comment]
Hi Nate @sestinj ,

I see you fixed the typo but it hasn't been pushed to the marketplace yet.

Thanks

**Comment by sestinj** [comment]
@zivshapiraintel  @elabbarw this is now released in 0.8.20 (https://github.com/continuedev/continue/releases/tag/v0.8.20-vscode) or the most recent pre-release. Thanks for the bump!

**Comment by sestinj** [comment]
@LeonKalt yes this fix will be included in the next release that we make
